### PR TITLE
get all cocktails

### PR DESCRIPTION
**Pull request by st0nebreaker**
# PR Description
Please include a summary of the change:
Moves API call so App will hold onto all cocktails array

Commit Emojis:
:recycle: refactor code.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [ ] My code follows the Turing style guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -7,17 +7,40 @@ import About from '../About/About';
...
+		idDrink: string
+}
+
 const App: React.SFC = () => {
 	const [ username, setUsername ] = useState('');
 	const [ loggedIn, setLoggedIn ] = useState(false);
-	//store favorite in app
-	//do all fetch calls & store info (look into useMemo)
-	//b/c child components will unmount & remount continuously but App never unmounts
+	const [ allCocktails, setAllCocktails ] = useState<AllCocktailsDetails[]>([{strDrink: '',
```
Comment (asodemann18) [inline src/App/App.tsx:22]: looks good!

File: src/apiCalls.tsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import { Cocktail } from "./Definitions/RandomCocktail";
-import { AllCocktailsDetails } from "./AllCocktailsPage/AllCocktailsPage";
+import { AllCocktailsDetails } from "./App/App";
 
 const rootUrl = "https://www.thecocktaildb.com/api/json/v1/1";
 
 //Add AlcoholicCocktail[] to definitions
-export const getAllCocktails = async (): Promise<AllCocktailsDetails> => {
+export const getAllCocktails = async (): Promise<AllCocktailsDetails[]> => {
```
Comment (asodemann18) [inline src/apiCalls.tsx:7]: Makes sense

**Review by asodemann18 — APPROVED** [review]
Looks good!

File: src/App/App.tsx [hunk 3]
```diff
@@ -7,17 +7,40 @@ import About from '../About/About';
...
+	const fetchAllCocktails = async (): Promise<any> => {
+    try {
+      const data: AllCocktailsDetails[] = await getAllCocktails();
+      return setAllCocktails(data);
+    } catch (error) {
+      setError(error.message);
+    }
+	};
+
+	useEffect(() => {fetchAllCocktails()}, []);
```
Comment (Melizzo) [inline src/App/App.tsx:39]: Good call, moving API calls to the App to do the heavy lifting

**Review by Melizzo — APPROVED** [review]

**Pull request merged**